src/components/input-panel/compiled-spec-header/index.tsx

```diff
@@ -62,7 +62,9 @@ class CompiledSpecDisplayHeader extends React.PureComponent<Props> {
             </li>
           </ul>
 
-          <ChevronDown />
+          <div className="down-button-svg">
```

Comment:
Why is this necessary?

Reply:
I tried `.editor-header svg` , its not working for me

Reply:
I'm not really satisfied by that answer.

Reply:
svg is an inline-element , we can place it inside div and position the div , otherwise if we want to do it with only SVG tag, we have to position it using pixels

Reply:
Really? Can't we just make it a block element for example?

Reply:
Tried! It's actually not working , maybe when we switch we remove some divs!
Np, I am trying to place it without the div , but I think this works good as well!